environment.yml

```diff
@@ -49,7 +49,7 @@ dependencies:
         - git+https://github.com/neocxi/Lasagne.git@484866cf8b38d878e92d521be445968531646bb8#egg=Lasagne
         - git+https://github.com/plotly/plotly.py.git@2594076e29584ede2d09f2aa40a8a195b3f3fc66#egg=plotly
         - awscli
-        - git+https://github.com/openai/gym.git@v0.7.4#egg=gym
```

Comment:
Have you tested that this still works by running training algorithms against it? Gym tends to have breaking API changes a lot.

Reply:
I have tested a few. They work. I will actually revert back to older version and try if Bullet works with that. I increased this version because of the comments from the other pull request. I will test this.

Reply:
I really want to upgrade it, I just want to make sure it didn't break anything.